testing/architecture-test/src/test/java/org/gradle/architecture/test/ArchUnitFixture.java

```diff
@@ -86,7 +86,8 @@ public interface ArchUnitFixture {
         "org.gradle.internal.serialize.graph..",
         "org.gradle.kotlin..",
         "org.gradle.internal.declarativedsl..",
-        "org.gradle.declarative.dsl.."
+        "org.gradle.declarative.dsl..",
+        "org.gradle.problems.internal.."
```

Comment:
🤔  Are all classes in the specified package written exclusively in Kotlin? If not, we risk losing coverage in all tests that use this predicate